src/outputs/total_fte.py

```diff
@@ -0,0 +1,48 @@
+"""The main file for the total FTE output."""
+import logging
+import pandas as pd
+from datetime import datetime
+from typing import Callable, Dict, Any
+
+
+OutputMainLogger = logging.getLogger(__name__)
+
+
+def qa_output_total_fte(
+    df: pd.DataFrame,
+    config: Dict[str, Any],
+    write_csv: Callable,
+    run_id: int
+):
+    """Run the outputs module.
+
+    Args:
+        df (pd.DataFrame): The main dataset with weights not applied
+        config (dict): The configuration settings.
+        write_csv (Callable): Function to write to a csv file.
+         This will be the hdfs or network version depending on settings.
+        run_id (int): The current run id
+
+    """
+    NETWORK_OR_HDFS = config["global"]["network_or_hdfs"]
+    est_path = config[f"{NETWORK_OR_HDFS}_paths"]["estimation_path"]
+
+    imp_marker_list = ["R", "TMI", "CR", "MoR"]
+    sliced_df = df.copy().loc[df["imp_marker"].isin(imp_marker_list)]
+
+    totals_names = ["Total",
+                    "Scientists and engineers",
+                    "Technicians, laboratory assistants and draughtsmen",
+                    "Administrative, clerical, industrial and other staff"]
```

Comment:
This is one of those cases where I wonder whether the requirements that Nick Powell wrote are actually needed, or whether  the business area would be happy with the actual column names we're using?

Reply:
@AnneONS let's use the names Nick suggested. The BaU might have to do less work renaming our techy acronyms to user-readable ones.